docs/sections/cn_workflows.md

```diff
@@ -417,64 +417,49 @@ by default.
+When a workflow is executed, Popper first creates a persistent volume claim, spawns an init pod and uses it to copy the workflow context (packed in the form of a `.tar.gz` file) into the persistent volume and then unpacks the context there. Then, popper teardowns the init pod and execute the steps of a workflow in separate pods of their own. After the execution of each step, the respective pods are deleted but the persistent volume claim is not deleted, so that it can reused by subsequent workflow executions.
 
-For running workflows on Kubernetes, some configuration options need to be passed to the kubernetes resource manager through the popper configuration file.
+For running workflows on kubernetes, several configuration options can be passed to the kubernetes resource manager through the popper configuration file to customize the execution environment. All the available configuration options has been described below.
 
-#### Running on a cluster without shared storage
+* `namespace`: The namespace within which to provision resources like PVCs and Pods for a workflow execution. If not provided the       `default` namespace will be used.
 
-**NOTE:** If your workflow needs to build an image from a `Dockerfile`, make sure you are logged in to dockerhub.
+* `persistent_volume_name`: Any pre-provisioned persistent volume like an NFS or EBS volume can be supplied through this option. Popper will then claim storage space from the supplied persistent volume. In the default case, a local persistent volume of 1Gi will be created by Popper automatically.
 
-1. Write a persistent volume defination similar to the one shown below.
+* `volume_size`: The amount of storage space to claim from a persistent volume for use by a workflow. Default is 500Mi.
 
-```bash
-$ cat<< EOF > pv.yaml
-kind: PersistentVolume
-apiVersion: v1
-metadata:
-  name: pv-hostpath
-  labels:
-    type: host
-spec:
-  persistentVolumeReclaimPolicy: Recycle
-  storageClassName: manual
-  capacity:
-    storage: 1Gi
-  accessModes:
-    - ReadWriteMany
-  hostPath:
-    path: "/tmp"
-EOF
-```
+* `pod_host_node`: The node on which to restrict the deployment of all the pods. This options is important when you are using a local persistent volume. In this case, users need to restrict all the pods to a particular node. If this option is not provided, Popper will leave the task of scheduling the pods upon kubernetes. The exception to this is, when both the `pod_host_node` and `persistent_volume_name` options are not provided, Popper will try to find out a schedulable pod and schedule all the pods (init-pods + step-pods) on that node in order to use the local persistent volume of 1Gi which will be automatically created.
```

Comment:
this has a statement in active voice ("you are using"), whereas the rest of the text is using the passive voice ("users need to restrict").